tests/testthat/_snaps/dataframe.md

```diff
@@ -387,3 +387,21 @@
       │ 5.0     ┆ 25.0 ┆ e   ┆ 1.0        ┆ 11.0         ┆ 0.0          │
       └─────────┴──────┴─────┴────────────┴──────────────┴──────────────┘
 
+# glimpse
+
+    Code
+      pl$DataFrame(mtcars)$with_columns(pl$lit(42)$cast(pl$Int8))$glimpse()
+    Output
+      & mpg    <f64>21, 21, 22.8, 21.4, 18.7, 18.1, 14.3, 24.4, 22.8, 19.2
+      & cyl    <f64>6, 6, 4, 6, 8, 6, 8, 4, 4, 6
```

Comment:
for readability, there should be a space between the type and the first value (but very minor)

Reply:
fair point :)